pull_config.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#
+# Pull config from the production database to the local database.
+#
+
+# Prompt the user before running
+echo
+echo "*******************"
+echo "**    WARNING    **"
+echo "*******************"
+echo
+
+read -p "You're about to update your local config database with the one on production. You sure about this? [y/n] " -r
```

Comment:
1. File change to `push_config.sh`: cool
2. Does it seem ok to leave this responsibility to the person making the changes for now? Eventually, we'll be working towards removing the config/*.json files from the repo entirely.
3. The jitsu restart - we do still need to do it for it take the config changes, right? A good goal for later that we talked about would be a way to push changes and not have to restart the app. Probably outside the scope of this PR though.
4. And specifying that mongorestore won't overwrite documents (I think mongoimport does the same): cool